src/SFA.DAS.ApplyService.Web/Controllers/Roatp/RoatpApplicationController.cs

```diff
@@ -172,41 +188,57 @@ private async Task<IActionResult> StartApplication(Guid signinId)
                 var allQnaSequences = await allQnaSequencesTask;
                 var allQnaSections = await allQnaSectionsTask;
 
-                var startApplicationRequest = await BuildStartApplicationRequest(qnaResponse.ApplicationId, user.Id, providerRoute, allQnaSequences, allQnaSections);
+                var startApplicationRequest = BuildStartApplicationRequest(qnaResponse.ApplicationId, user.Id, providerRoute, allQnaSequences, allQnaSections);
 
                 var applicationId = await _apiClient.StartApplication(startApplicationRequest);
                 _logger.LogInformation($"RoatpApplicationController.StartApplication:: Checking response from StartApplication POST: applicationId: [{applicationId}]");
 
                 if (applicationId != Guid.Empty)
                 {
                    await SavePreambleInformation(applicationId, applicationDetails);
+                   _logger.LogInformation("Preamble information saved");
 
                     if (applicationDetails.UkrlpLookupDetails.VerifiedByCompaniesHouse)
                     {
                         await SaveCompaniesHouseInformation(applicationId, applicationDetails);
+                        _logger.LogInformation("Companies House information saved");
                     }
 
                     if (applicationDetails.UkrlpLookupDetails.VerifiedbyCharityCommission)
                     {
                         await SaveCharityCommissionInformation(applicationId, applicationDetails);
+                        _logger.LogInformation("Save Charity Commission information saved");
                     }
                 }
+                
+                _logger.LogInformation("StartApplication method completed");
```

Comment:
As mentioned above, could these be downgraded to LogDebug?

Reply:
Changed to debug